src/Microsoft.DotNet.GitHub.IssueLabeler/Models/Labeler.cs

```diff
@@ -22,51 +23,66 @@ public class Labeler
         private GitHubClient _client;
         private HttpClient _httpClient;
         private Regex _regex;
-        private readonly string _repoOwner;
-        private readonly string _repoName;
         private readonly double _threshold;
         private readonly string _secretUri;
+        private readonly bool _skipAzureKeyVault;
         private readonly DiffHelper _diffHelper;
-        private readonly DatasetHelper _datasetHelper;
         private readonly string MessageToAddDoc =
             "Note regarding the `new-api-needs-documentation` label:" + Environment.NewLine + Environment.NewLine +
             "This serves as a reminder for when your PR is modifying a ref *.cs file and adding/modifying public APIs, to please make sure the API implementation in the src *.cs file is documented with triple slash comments, so the PR reviewers can sign off that change.";
-        private readonly string MessageToAddAreaLabelForPr =
-            "I couldn't figure out the best area label to add to this PR. Please help me learn by adding exactly one " + AreaLabelLinked + ".";
-        private readonly string MessageToAddAreaLabelForIssue =
-            "I couldn't figure out the best area label to add to this issue. Please help me learn by adding exactly one " + AreaLabelLinked + ".";
-        private static readonly string AreaLabelLinked =
-            "[area label](" + 
+        private string _messageToAddAreaLabelForPr =>
+            "I couldn't figure out the best area label to add to this PR. If you have write-permissions please help me learn by adding exactly one " + _areaLabelLinked + ".";
+        private string _messageToAddAreaLabelForIssue =>
+            "I couldn't figure out the best area label to add to this issue. If you have write-permissions please help me learn by adding exactly one " + _areaLabelLinked + ".";
+        private string _areaLabelLinked =>
+            RepoName.Equals("runtime", StringComparison.OrdinalIgnoreCase) ? "[area label](" + 
```

Comment:
we should be able to learn from one area label and some other labels which are not labels like eg if other label is "test -bug"